modules/@apostrophecms/schema/index.js

```diff
@@ -1155,11 +1155,11 @@ module.exports = {
 
         // always make sure there is a default group
         const defaultGroup = self.options.defaultGroup || {};
-        let groups = [{
-          name: defaultGroup.name || 'default',
-          label: defaultGroup.label || 'Info',
+        let groups = [ {
+          name: defaultGroup.name || 'other',
```

Comment:
I'm differentiating here between default fields and unarranged fields. Maybe this should be `unarranged` for the name property instead?

Reply:
`unarranged` sounds smart.

Reply:
I'm sure people with put things in this intentionally at some point. I guess that's my hesitation at the moment.